tools/rtpsrelay/RelayHandler.h

```diff
@@ -108,9 +110,13 @@ class VerticalHandler : public RelayHandler {
   void process_message(const ACE_INET_Addr& remote,
                        const OpenDDS::DCPS::MonotonicTimePoint& now,
                        const OpenDDS::DCPS::Message_Block_Shared_Ptr& msg) override;
-  void send(const RelayAddressesMap& relay_addresses_map,
+  void send(const GuidSet& to,
             const OpenDDS::DCPS::Message_Block_Shared_Ptr& msg);
+  void populate_relay_addresses_map(RelayAddressesMap& relay_addresses_map,
```

Comment:
Would this be cleaner to just have the `RelayAddressesMap` as the return type?  We can now assume move semantics and probably some level of copy elision.